src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -89,4 +89,17 @@
         btn.dataset.isFormValid = 0;
         btn.addEventListener('click', clickHandler, false);
     });
+
+    global.addEventListener('scroll', () => {
+        doc.getElementById('ez-edit-header-details').style.maxHeight = window.scrollY > 70 ? '0px' : '70px';
```

Comment:
I don't like the idea of having `scroll` listener - it's costly. 
Can be done in pure CSS. Fixed should be only the top of the header with content name not all informations.